rex/temporal_stats/temporal_stats.py

```diff
@@ -603,6 +626,10 @@ def compute_statistics(self, dataset, sites=None, diurnal=False,
             Number of chunks to extract on each worker, by default 5
         lat_lon_only : bool, optional
             Only append lat, lon coordinates to stats, by default True
+        mask_zeros : bool
+            Flag to only calculate stats when all data is > 0 (useful for
+            global horizontal irradiance). This also causes each site to be
+            processed individually.
```

Comment:
Should we make it clear (here and everywhere else lol) that sites with all 0 values are dropped completely from the output? It's implied here but without looking at the code I wasn't sure if they would all be set to 0 or what

Reply:
I think they would actually get NaN, but im not too worried about this. We should never really have a full timeseries of 0 resource.